src/lib.rs

```diff
@@ -669,7 +671,18 @@ pub struct CameraAnimationOptions {
     #[serde(flatten)]
     animation_options: AnimationOptions,
 }
-
+#[wasm_bindgen]
+extern "C" {
+    // Use `js_namespace` here to bind `console.log(..)` instead of just
+    // `log(..)`
+    #[wasm_bindgen(js_namespace = console)]
+    fn log(s: &str);
+}
+macro_rules! console_log {
+    // Note that this is using the `log` function imported above during
+    // `bare_bones`
+    ($($t:tt)*) => (log(&format_args!($($t)*).to_string()))
+}
```

Comment:
nice!

Reply:
nice but not actually used :see_no_evil: It was for debugging.